crates/torii/core/src/processors/event_message.rs

```diff
@@ -72,7 +73,8 @@ where
         entity.deserialize(&mut keys_and_unpacked)?;
 
         // TODO: this must come from some torii's configuration.
-        let historical = false;
+        let historical =
```

Comment:
To make it more effective, we should compute the selector when torii loads the configuration, like so we only have 1 felt to compare.

But not blocking at all and can be done later. 👍